cluster/addons/cluster-monitoring/influxdb/influxdb-grafana-petset.yaml

```diff
@@ -64,9 +65,21 @@ spec:
           - name: grafana-persistent-storage
             mountPath: /var
       volumes:
-      - name: influxdb-persistent-storage
-        persistentVolumeClaim:
-          claimName: influxdb-claim
       - name: grafana-persistent-storage
         emptyDir: {}
-  serviceName: monitoring-influxdb
+  volumeClaimTemplates:
+  - metadata:
+      name: influxdb-persistent-storage
+      namespace: kube-system
+      #labels:
+        #kubernetes.io/cluster-service: "true"
+      annotations:
+        volume.alpha.kubernetes.io/storage-class: "anything"
```

Comment:
Storage class will auto provision, if you don't want that you should not set the annotation.